### DISCUSSION TITLE
Difficulties uploading a file using API

### DISCUSSION THREAD

**Discussion by kropsi** [directus/directus#14292] [Archive]
I am trying to use [Upload a File API](https://docs.directus.io/reference/files/#upload-a-file), but no luck so far:

![CleanShot 2022-07-05 at 16 19 53@2x](https://user-images.githubusercontent.com/946662/177349985-3b4be057-cc45-44b1-957e-d61ad4c0c929.png)

No matter what I do (Directus 9.13.0) I am not able to upload a file using Postman.

There is access to Files
<img width="802" alt="CleanShot 2022-07-05 at 16 23 49@2x" src="https://user-images.githubusercontent.com/946662/177350744-9d056056-2433-4ecf-b633-af8e95057de5.png">

Please share your experience with this, I hope you can help.

**Comment by rijkvanzanten** [comment]
What is the 500 thrown in the API? Any logs in the process?

**Reply by kropsi** [reply]
I am not sure that I understand your first question. Where can I see logs? Is there a way to catch errors in logs somewhere? There is no logs folder i directus folder on server.

**Reply by rijkvanzanten** [reply]
Wherever you're running the Directus process (eg `directus start`) is where Directus will show logs by default 🙂

**Reply by kropsi** [reply]
Yes, thank you @rijkvanzanten I was able to identify the problem which was with this multipart file upload. I am running in App Platform (Digital Ocean) and there is a Runtime log, which I found.

**Reply by rijkvanzanten** [reply]
What was the problem and solution? Might be helpful for others running into the same thing 🙂

**Reply by kropsi** [reply]
I am integrating some Directus APIs using Make.com and the problem was that the body type for integration was not specified in Make platform, it should be set to: multipart/form-data. I was not able to detect that as Make doesn't output the reason for that erorr. I believe it is a bit platform specific, but nevertheless I succeeded implementing Upload a file module in my Make.com app, so it works perfectly now :)
